examples/blender/blender.py

```diff
@@ -27,7 +27,7 @@
 script_dir = pathlib.Path(__file__).resolve().parent
 
 
-async def main(subnet_tag: str):
+async def main(subnet_tag):
```

Comment:
This change is required since when `examples` becomes a package, `mypy` starts checking the examples and reports some errors (actually, just one). Removing the annotation makes `mypy` ignore this function.